test/integration/repos_test.rb

```diff
@@ -12,4 +15,34 @@ class ReposTest < ActionDispatch::IntegrationTest
       get '/tgxworld/rails'
     end
   end
+
+  test "query JSON endpoint should return JSON object" do
+    benchmark_type = create(:benchmark_type)
+    benchmark_result_type = create(:benchmark_result_type)
+
+    repo = benchmark_type.repo
+    org = repo.organization
+
+    commit = create(:commit, repo: repo)
+    later_commit = create(:commit, repo: repo, created_at: Time.zone.now + 1.day)
+
+    benchmark_run = create(:commit_benchmark_run,
+      benchmark_result_type: benchmark_result_type,
+      benchmark_type: benchmark_type,
+      initiator: commit
+    )
+
+    benchmark_run2 = create(:commit_benchmark_run,
+      benchmark_result_type: benchmark_result_type,
+      benchmark_type: benchmark_type,
+      initiator: later_commit
+    )
+
+    get "/#{org.name}/#{repo.name}/commits.json?result_type=#{benchmark_type.category}", 
+      params: { display_count: 2, result_type: benchmark_result_type }
+
+    chart = JSON.parse(response.body, symbolize_names: true)[0]
+    # must have these 4 keys
+    assert ([:benchmark_name, :datapoints, :measurement, :unit] - chart.keys).empty?
```

Comment:
```ruby
assert_include :benchmark_name, chart.keys
assert_include :datapoints, chart.keys
...
```

seems like a more appropriate way as it requires less thought as to what's going on